index.js

```diff
@@ -34,8 +37,18 @@ function updateUi() {
 }
 
 function loadTopoJson(topoJson) {
+  importing = true
   const tiles = importer.fromTopoJson(topoJson)
-  canvas.importTiles(tiles)
+  const datasetMap = {}
+  tiles.forEach((tile) => {
+    datasetMap[tile.id] = [tile.id, tile.tilegramValue]
+  })
+  const dataset = Object.keys(datasetMap).map((row) => datasetMap[row])
```

Comment:
Could we put the above few lines into a helper in `Data` called like `buildDatasetFromTiles()` or something?